src/RawPicture.cpp

```diff
@@ -18,64 +18,63 @@
  *
  */
 
-#include "kodi_imagedec_dll.h"
-
+#include <kodi/addon-instance/ImageDecoder.h>
 #include <libraw.h>
-#if defined(TARGET_WINDOWS) && defined(LoadImage)
-#undef LoadImage
-#endif
 
-class RawPicture
+class RawPicture : public kodi::addon::CInstanceImageDecoder
 {
 public:
-  RawPicture()
+  RawPicture(KODI_HANDLE instance)
+    : CInstanceImageDecoder(instance),
+      m_raw_data(libraw_init(0)),
+      m_thumbnailbuffer(nullptr)
```

Comment:
Have looked not used :), there is also a function call with `libraw_unpack_thumb` for where it is needed?